### DISCUSSION TITLE
I saw that This app cannot validate signatures yet.

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#188] [Q&A] [answered]
As the title says, i saw that.
What's meaning??
Just i do personal_sign

![image](https://github.com/WalletConnect/WalletConnectUnity/assets/109068040/348a7114-b6cc-4421-a363-e5c56a9dccbd)

**Comment by skibitsky** [comment, marked as answer]
The wallet returns a signed message as the response. To validate that this message is valid (i.e. signed by the address of the connected wallet), you need to use a third-party library such as Nethereum.

[Verifying a Signed Message Encoded in UTF8 Using EncodeUTF8AndEcRecover](https://docs.nethereum.com/en/latest/nethereum-signing-messages/#2-verifying-a-signed-message-encoded-in-utf8-using-encodeutf8andecrecover)

**Reply by matthew2099** [reply]
Ah i see. 
I check that.
Just i use sample and I can see that..hum

**Reply by skibitsky** [reply]
I see. The sample is only to show the WalletConnect part, which is dapp<>wallet communication. Encoding, verification, etc., should be handled by the dapp developer with other libraries.

**Reply by matthew2099** [reply]
@skibitsky 
I understand. Really thank you.
